src/api/center.js

```diff
@@ -15,12 +15,14 @@ const center = (objects, options) => {
 
   const results = objects.map(function (object) {
     let b = object.getBounds()
-    return object.translate(axes.map(function (a) {
-      return cAxes.indexOf(a) > -1 ? -(b[0][a] + b[1][a]) / 2 : 0
-    }))
+    let offset = [0,0,0]
+    if (axes[0]) offset[0] = 0 - (b[0]._x + ((b[1]._x - b[0]._x) / 2))
```

Comment:
seems good to me !
but why use the _x & _y etc instead of just x & y ? (is it the whole accessors issue?)